doc/stdenv/stdenv.chapter.md

```diff
@@ -1133,164 +1140,6 @@ Similarly, the CC Wrapper follows the Bintools Wrapper in defining standard envi
 
 Here are some more packages that provide a setup hook. Since the list of hooks is extensible, this is not an exhaustive list. The mechanism is only to be used as a last resort, so it might cover most uses.
 
```

Comment:
This should be replaced by the `### Other hooks` section.

Reply:
These are the lines that have been removed and added to the chapter referenced by Other hooks.
You might have misinterpreted the diff.